modules/c/pages/gs-install.adoc

```diff
@@ -289,6 +310,51 @@ Then, within the dialog:
 
 {install-closure}
 
+=== Install Vector Search for macOS
+
+. Download and extract the link:https://www.couchbase.com/downloads/?family=couchbase-lite#LiteExtensions[Vector Search extension] into your XCode project location.
+. Select your target settings in XCode and drag *CouchbaseLiteVectorSearch.xcframework* from your Finder to the *Frameworks, Libraries, and Embedded Content* section.
+. Import the xcframework and start using it in your project.
+. Find the extension path using the appropriate snippet:
```

Comment:
The step is similar to the step to install CBL dylib but the differences are :

1. We don't support homebrew. We only package the zip file.

2. It will be CouchbaseLiteVectorSearch.dylib.

3. Developer must download and install the vector search library to the location in your project where the vector search library can be accessed and loaded while the executable is running. (Same as the linux platform - it will depend on the developer where to put the vector search library for the application.).

Reply:
I think I've done this